core/src/layers/throttle.rs

```diff
@@ -221,24 +220,22 @@ impl<R: oio::Write> oio::Write for ThrottleWrapper<R> {
 
         loop {
             match self.limiter.check_n(buf_length) {
-                Ok(_) => return self.inner.poll_write(cx, bs),
-                Err(negative) => match negative {
+                Ok(res) => match res {
```

Comment:
Can you add a brief for the change here? I know that it's caused by governor's breaking change, but how do you handle it?

Reply:
It just unfriendly for a reviewer to read this complex logic and reverse engineering what you do.